util.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+# Install deps
+dnf -y install jq dosfstools
+
```

Comment:
can you drop the install here, this was added in https://github.com/containers/podman-machine-os/pull/34

Reply:
done